static/fix-missing-vars.js

```diff
@@ -1,2 +1,3 @@
 const process = {env:{}}
-const global = {}
\ No newline at end of file
+const global = {}
+const development = 'production'
```

Comment:
I'm concerned about this @Simon-Laux what do you think? In theory this should change depending on production or development build.

Reply:
why did you add it here?

Reply:
That was what i was looking for. Didn't find it when i was scrolling through the esbuild docs :)